compiler/rustc_trait_selection/src/solve/eval_ctxt.rs

```diff
@@ -548,4 +567,56 @@ impl<'tcx> EvalCtxt<'_, 'tcx> {
     pub(super) fn universe(&self) -> ty::UniverseIndex {
```

Comment:
also feel like `universe` shouldn't really matter for any behavior of the `EvalCtxt`, so having that unavailable would also be nice